### PR TITLE
[Serial API article] Add launched info

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR makes sure the article reflects the Serial API launching in Chrome 89.

Live preview:
- https://deploy-preview-4436--web-dev-staging.netlify.app/serial
- https://deploy-preview-4436--web-dev-staging.netlify.app/fugu-status

@reillyeon FYI

### CODE REVIEW

**Review by reillyeon — APPROVED** [review]

File: src/site/content/en/blog/serial/index.md [hunk 1]
```diff
@@ -4,15 +4,13 @@ subhead: The Serial API allows websites to communicate with serial devices.
 authors:
   - beaufortfrancois
 date: 2020-08-12
-updated: 2020-12-02
+updated: 2021-01-06
```
Comment (jpmedley) [inline src/site/content/en/blog/serial/index.md:7]: I believe these dates affect RSS feeds. Please update these before going live.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/serial/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/serial/index.md:7]: I will.

**Review by kaycebasques — APPROVED** [review]

**Comment by jpmedley** [comment]
@beaufortfrancois There are 3 LGTMs on the I2S and the last commit is merged to the 89 branch. Is there any reason not to go ahead and publish this?

**Comment by beaufortfrancois** [comment]
You're right. I'll go ahead and merge this PR.

File: src/site/content/en/blog/serial/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/serial/index.md]: I've updated it.

**Pull request merged**